example/lib/main.dart

```diff
@@ -93,20 +93,31 @@ class MinionComponent extends FlareActorComponent with Resizable {
   MinionController minionController;
 
   MinionComponent(this.minionController)
-      : super(FlareActorAnimation(
-          'assets/Bob_Minion.flr',
-          controller: minionController,
-          fit: BoxFit.contain,
-          alignment: Alignment.bottomCenter,
-          width: 306,
-          height: 500,
-        ));
+      : super(
+          FlareActorAnimation(
+            'assets/Bob_Minion.flr',
+            controller: minionController,
+            fit: BoxFit.scaleDown,
+            alignment: Alignment.center,
+          ),
+          width: 300,
+          height: 300,
+        );
+
+  @override
+  void render(Canvas c) {
```

Comment:
Suggestion: If you mark the basegame as debugmode true (check the docs on flame on how to) the bounding box will be drawn automatically

Reply:
Yep, it is actvated